autonomy/deploy/generators/kubernetes/templates.py

```diff
@@ -281,6 +281,17 @@
         - name: nodes
           persistentVolumeClaim:
             claimName: 'nodes'
+        - emptyDir: {{}}
+          name: local-tendermint
+      initContainers:
+        - name: copy-tendermint-configuration
+          image: "ubuntu/ubuntu:20.04"
+          command: ["cp", "-r", "/tendermint/node{validator_ix}", "/tm"]
```

Comment:
this doesn't match above?

Reply:
We are doing a bit of juggling in the init container.

init container starts with 2 volume mounts;
local-tendermint -> this is a host path pv.
nodes -> where the config get written to.

We copy the directory when it is available, and, additionally we dont copy over all of the configs, only the oone relevant for our use.